apps/client/src/features/modals/AppSettingsModal.jsx

```diff
@@ -128,6 +133,24 @@ export default function AppSettingsModal() {
     setChanged(true);
   };
 
+  /**
+   * Handles version comparison and returns text
+   */
+  const versionString = () => {
+    //remove v. added by github
+    const latestVersion = versionData.version.substring(2);
+    const installedVersion = version;
+    if (latestVersion === installedVersion) {
+      const text = `Running latest version: ${installedVersion}`;
+      return text;
+    } else {
+      const text = (
+        <a href={versionData.url} target='_blank' rel='noreferrer'>{`New version available: ${latestVersion}`}</a>
+      );
+      return text;
+    }
+  };
+
```

Comment:
we can simplify this a little by making some assumptions.
if the version in github is v1.1.1 and the local is 1.1.1, I know that for them to be the same, the github one has to contain the local

Also, 
- redeclaring constants in collocated scopes can cause hard-to-debug issues in future, it would be simpler and more dummy proof to immediately return
- we probably want to make sure to always return the same data type. By Calling a function named versionString() I would expect to get a string representation of my app version, not a component